examples/android/LuminaDemo/README.md

```diff
@@ -0,0 +1,18 @@
+# Instructions to re-create the setup
+
+- Create a new kotlin project in xcode
+- make sure you have aarch64-apple-ios target installed + simulator
+```
+rustup target add aarch64-linux-android armv7-linux-androideabi i686-linux-android thumbv7em-none-eabihf wasm32-unknown-unknown x86_64-linux-android
```

Comment:
why is wasm needed?

Reply:
It's not, I've copied extra targets from my installed target list by accident. Removed